src/main/java/ch/hsr/geohash/util/DoubleUtil.java

```diff
@@ -0,0 +1,17 @@
+package ch.hsr.geohash.util;
+
+public class DoubleUtil {
+
+	/**
+	 * Utility function to consistently compute the remainder over all java versions
+	 * @param value
+	 * @param remainder
+	 * @return
+	 */
+	public static double remainderWithFix(double value, int remainder) {
+		double res = value % remainder;
+		
+		// Fix for lower java versions, since the remainder-operator (%) changed in one version, idk which one
```

Comment:
> 
> 
> I don't really get this. Which version of java would've had modulus implemented differently? Also the value of `res` is never really used in one branch.

I tried to do a quick research but I didn't find the corresponding versions. I can dig more into it if you wish so.

A colleague has a mathematical background and told me that in mathematics the result of the modulo operation is always positiv.
This behaviour is implemented differently in different programming languages, e.g. [wiki](https://en.wikipedia.org/wiki/Remainder) (section Programming languages). If I remember correctly somewhere around jdk8 (is a best guess right now, has been a while) they changed the behaviour to always return positiv remainders. Atleast I tested it with different jdk versions and it did not always work.

The intent is indeed to clamp the value to `(0,360)`.

Reply:
> Also the value of res is never really used in one branch.

Can you explain this one to me? If I look at this line
```java
return res < 0 ? res += remainder : res;
```
`res` is used in both branches.

Reply:
Be it as it may, the function works as intended, but I would love to rename it to `clamp()` and change the parameters to be `clamp(value, min, max)` for clarity.

Reply:
Well, actually it doe not really clamp, but rather something like `fold` into a range. Let me think about a good name for a sec. :)

Reply:
Sorry for the confusion there. Let me push this.

Reply:
Ok I'm waiting 😄 Take your time.

Reply:
We could just call it `positiveRemainder` but idk if that would be intuitive enough.
For me I'd work but that's no reference 😅

Reply:
Hey @kungfoo did you come up with a good name yet?

Reply:
Let's go with this name for now, as it is only used internally.